django/contrib/admin/options.py

```diff
@@ -858,9 +858,13 @@ def action_checkbox(self, obj):
     def _get_base_actions(self):
         """Return the list of actions, prior to any request-based filtering."""
         actions = []
+        base_actions = [self.get_action(action) for action in self.actions or []]
+        base_action_names = {action[1] for action in base_actions if action}
```

Comment:
How about:

```py
base_action_names = {name for _, name, _ in filter(None, base_actions)}
```

I think it's more readable because we'll know that `base_actions` is a list of 3-tuple which the second element is the name (and the other two is ignored).